gdcdictionary/schemas/clinical_trials.yaml

```diff
@@ -124,44 +132,49 @@ properties:
       - "Phase II/III"
       - "Phase N/A"
 
-  clinical_trials:
```

Comment:
breaking change - do we have no existing data with the `clinical_trials` prop?

Reply:
We don't have any existing data with this prop. No data has been submitted to the clinical trials node yet.